packages/wdio-browserstack-service/src/util.ts

```diff
@@ -596,7 +596,7 @@ export const stopAccessibilityTestRun = errorHandler(async function stopAccessib
     }
 })
 
-export const stopBuildUpstream = o11yErrorHandler(async function stopBuildUpstream() {
+export const stopBuildUpstream = o11yErrorHandler(async function stopBuildUpstream(killSignal: string|null = null) {
```

Comment:
optional arg

Reply:
we dont need it here since we are setting a default value